demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+                memory: 128Mi
+              limits:
+                cpu: 250m
+                memory: 256Mi
+            ports:
+            - containerPort: 80
+            env:
+            - name: REDIS
+              value: "azure-vote-back"
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: azure-vote-front
+    spec:
+      type: LoadBalancer
+      ports:
+      - port: 80
+      selector:
+        app: azure-vote-front
+
+    EOF
+    ```
+      For a breakdown of YAML manifest files, see [Deployments and YAML manifests](../concepts-clusters-workloads.md#deployments-and-yaml-manifests).
+
+1. Deploy the application using the [kubectl apply][kubectl-apply] command and specify the name of your YAML manifest:
+
+    ```azurecli-interactive
+    kubectl apply -f azure-vote.yaml
+    ```
+
+    The following example resembles output showing the successfully created deployments and services:
+    <!--expected_similarity=0.8-->
+    ```output
+    deployment "azure-vote-back" created
+    service "azure-vote-back" created
+    deployment "azure-vote-front" created
+    service "azure-vote-front" created
+    ```
+
+## Test the application
+
+When the application runs, a Kubernetes service exposes the application front-end to the internet. This process can take a few minutes to complete.
+
+Monitor progress using the [kubectl get service][kubectl-get] command.
+
+```azurecli-interactive
+timeout 30 kubectl get service azure-vote-front --watch
+```
+
+The **EXTERNAL-IP** output for the `azure-vote-front` service will initially show as *pending*.
+<!--expected_similarity=0.4-->
+```output
+NAME               TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+azure-vote-front   LoadBalancer   10.0.37.27   <pending>     80:30572/TCP   6s
+```
+
+Once the **EXTERNAL-IP** address changes from *pending* to an actual public IP address, use `CTRL-C` to stop the `kubectl` watch process. The following example output shows a valid public IP address assigned to the service:
+
+```output
+azure-vote-front   LoadBalancer   10.0.37.27   52.179.23.131   80:30572/TCP   2m
+```
+
+To see the Azure Vote app in action, open a web browser to the external IP address of your service.
+
+:::image type="content" source="media/quick-kubernetes-deploy-portal/azure-voting-application.png" alt-text="Screenshot of browsing to Azure Vote sample application.":::
+
+## Clean up resources
+
+To avoid Azure charges, if you don't plan on going through the tutorials that follow, clean up your unnecessary resources. Use the [az group delete][az-group-delete] command to remove the resource group, container service, and all related resources.
+
+```azurecli-interactive
+az group delete --name $RESOURCE_GROUP_NAME --yes --no-wait
+```
+
+> [!NOTE]
+> The AKS cluster was created with system-assigned managed identity (default identity option used in this quickstart), the identity is managed by the platform and does not require removal.
```

Comment:
What about the second RG that it created, will that be cleaned up too?

Reply:
It is cleaned up, but that would be good to specify.